.github/workflows/aws.yml

```
@@ -0,0 +1,94 @@
+
+permissions:
+  contents: read
+
+jobs:
+  deploy:
+    name: Deploy
+    runs-on: ubuntu-latest
+    environment: production
+
+    steps:
+    - name: Checkout
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 6 but found 4

(indentation)